ccmlib/dse_node.py

```diff
@@ -95,6 +95,8 @@ def start(self,
         if wait_other_notice:
             marks = [(node, node.mark_log()) for node in list(self.cluster.nodes.values()) if node.is_running()]
 
+        self.mark = self.mark_log()
```

Comment:
Any particular reason to make this `self.mark` and not just `mark`? I worry about random state on a started node being an attractive nuisance.